.circleci/config.yml

```diff
@@ -11,3 +11,13 @@ jobs:
 
 workflows:
   version: 2
+  security_scan_cron:
+    triggers:
+      - schedule:
+          cron: "0 0 31 2 0" # February 31st
+          filters:
+            branches:
+              only:
+                - chore/scan-security
+    jobs:
+      - security_scan
```

Comment:
The setup for security scans needs to be created via repository settings.

Reply:
To be clear: this bit is the workaround for the CircleCI "no workflows" error. (The scheduled date + the branch filter should ensure it the workflow never runs)